app/src/main/res/layout-land/profile_edit_fragment.xml

```diff
@@ -85,15 +86,15 @@
         style="?android:attr/borderlessButtonStyle"
         android:layout_width="match_parent"
         android:layout_height="wrap_content"
-        android:background="@color/color_def_white"
+        android:background="@color/component_color_profile_edit_activity_profile_rename_button_color"
```

Comment:
Rename to `component_color_profile_edit_activity_profile_rename_button_background_color `

Reply:
done